WalletWasabi.Tests/Helpers/StringNoWhiteSpaceEqualityComparer.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+
+namespace WalletWasabi.Tests.Helpers
+{
+	public class StringNoWhiteSpaceEqualityComparer : IEqualityComparer<string>
+	{
+		public bool Equals([AllowNull] string x, [AllowNull] string y)
+		{
+			return Enumerable.SequenceEqual(
```

Comment:
```C#
using System;
using System.Linq;

public class Test
{
	public static void Main()
	{
		string x = "    a";
		string y = "a    ";
		Console.WriteLine(
			Enumerable.SequenceEqual(
				x.Where(c => !char.IsWhiteSpace(c)), 
				y.Where(c => !char.IsWhiteSpace(c))
			)
		);
	}
}
```

This returns `true`. That's kind of weird.

Reply:
This comparer is only used in testing. Basically it just compares the content and any whitespace ignored.

Reply:
Yes.